lgtm.yml

```diff
@@ -14,6 +14,8 @@ extraction:
       - libpython2.7-dev
       - libgtest-dev
       - dh-exec
+      - libmzq3-dev
```

Comment:
libzmq3?

Reply:
dev is with version 3, build is in version 5
https://packages.ubuntu.com/search?keywords=libzmq&searchon=names&suite=focal&section=all

Reply:
ok noticed mistake in name